OTAnalytics/plugin_ui/application.py

```diff
@@ -1,48 +1,112 @@
 import tkinter
+from abc import abstractmethod
 from pathlib import Path
 
 import customtkinter
 from customtkinter import CTk, CTkButton
 
 from OTAnalytics.application.datastore import Datastore
-from OTAnalytics.plugin_parser.otvision_parser import OttrkParser
+from OTAnalytics.domain.section import Coordinate, LineSection, Section
+from OTAnalytics.plugin_parser.otvision_parser import (
+    OtsectionParser,
+    OttrkParser,
+    OttrkVideoParser,
+)
 
 
 class OTAnalyticsApplication:
     def __init__(self) -> None:
-        self.datastore: Datastore
-        self.app: CTk
+        self._datastore: Datastore
```

Comment:
Class variable is not assigned yet. This will raise an error for as soon as `self._datastore` is accessed either in this `class or it's concrete classes.